controls/3.7/update_def.cf

```diff
@@ -48,6 +48,11 @@ bundle common update_def
       ifvarclass => not(isvariable("override_data_input_name_patterns")),
       meta => { "defvar" };
 
+      "masterfiles_location"
+        string => "/var/cfengine/masterfiles",
+        handle => "common_update_def_vars_default_masterfiles_location",
```

Comment:
This is the appropirate path for most environments, intended to be instrumented for overriding from the augments_file.